symbols/arm9.yml

```diff
@@ -9998,6 +9998,11 @@ arm9:
         
         r0: Dungeon ID
         return: Dungeon mode
+    - name: SoundUtilGetRandomNumber
+      address:
+        EU: 0x206CC8C
+        NA: 0x206C8F4
+      description: "return: random number"
```

Comment:
Do we know the range of random numbers possible?

Reply:
Looks like [0, 32767] based on the ASM. I'll add this to the description.